### PR TITLE
LPS-174340 Add field in testrayTask json

### PR DESCRIPTION
**Pull request by Elvinho**

### CODE REVIEW

**Comment by Elvinho** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html><details><summary>More details about: <b>LPS-174340 Improvements in testflow and case results</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-174340">Improvements in testflow and case results</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Task</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Minor</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>In Progress</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Elvison Victor</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Elvison Victor</td>
        </tr>
                </table>
          </details></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

9 files changed, 127 insertions(+), 56 deletions(-)

**Review by samuelleolopes — APPROVED** [review]
Good Job! :fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -13,15 +13,18 @@
...
 import React from 'react';
 
 type AvatarProps = {
 	className?: string;
 	displayName?: boolean;
+	displayTooltip?: boolean;
 	expanded?: boolean;
 	name?: string;
 	size?: 'lg' | 'sm' | 'xl';
+	tooltipPosition?: string;
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Avatar/Avatar.tsx:27]: Change the Type to correctly type

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx [hunk 2]
```diff
@@ -68,6 +67,7 @@ const TestFlowTasks = () => {
 		data: {testrayTask, testrayTaskUser},
 		revalidate: {revalidateSubtask},
 	} = useOutletContext<OutletContext>();
+
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx:70]: remove this  blank line.

**Review by rafaelfs98 — APPROVED** [review]
LGTM :fire:

**Review by welitonpe — CHANGES_REQUESTED** [review]
Good job, but you need to make some little bit changes.

**Review by caiosouza15 — APPROVED** [review]
LGTM :fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Tooltip/index.tsx [hunk 3]
```diff
@@ -14,9 +14,19 @@
...
 
+export type ALIGN_POSITIONS =
+	| 'top'
+	| 'top-left'
+	| 'top-right'
+	| 'bottom'
+	| 'bottom-left'
+	| 'bottom-right'
+	| 'left'
+	| 'right';
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Tooltip/index.tsx:25]: @Elvinho can you order it?

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/index.tsx [hunk 4]
```diff
@@ -146,22 +145,31 @@ const TestFlow = () => {
...
 					],
 					navigateTo: (item) => `/testflow/${item.id}`,
 					rowWrap: true,
 				}}
-				transformData={(response) =>
-					testrayTaskImpl.transformDataFromList(response)
-				}
+				transformData={(response) => {
+					return testrayTaskImpl.transformDataFromList(response);
+				}}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/index.tsx:172]: Simplify

```suggestion
				transformData={(response) => 
				     testrayTaskImpl.transformDataFromList(response)
				}
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
Good job on this @Elvinho , I left few suggestions for you

**Comment by Elvinho** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**